### PR TITLE
🩹(FIX) : Updated children's aupair field when hiring and terminating

### PR DESCRIPTION
**Pull request by PxttytheThxtty**

### CODE REVIEW

**Review by priyo5 — APPROVED** [review]

File: libs/hire-requests/feature/src/lib/hire-requests.component.ts [hunk 1]
```diff
@@ -210,6 +221,31 @@ export class HireRequestsComponent implements OnInit {
     )
   }
 
+  async getChildrenDetails(parent : string)
```
Comment (ricecakesreece) [inline libs/hire-requests/feature/src/lib/hire-requests.component.ts:224]: Rename function to be more descriptive e.g "setChildrenAuPair"

File: libs/parent-dashboard/feature/src/lib/parent-dashboard.ts [hunk 2]
```diff
@@ -296,6 +307,31 @@ export class ParentDashboardComponent implements OnInit{
     )
   }
 
+ async getChildrenDetails()
```
Comment (ricecakesreece) [inline libs/parent-dashboard/feature/src/lib/parent-dashboard.ts:310]: Rename function to be more descriptive e.g "removeChildrenAuPair"

**Review by Ruben-Brits — DISMISSED** [review]

File: libs/au-pair-dashboard/feature/src/lib/au-pair-dashboard.component.ts [hunk 3]
```diff
@@ -184,4 +223,182 @@ export class AuPairDashboardComponent implements OnInit {
...
+    const then  = new Date(this.currentAuPair.terminateDate);
+    const now = new Date();
+
+    const msBetweenDates = Math.abs(then.getTime() - now.getTime());
+
+    let daysBetweenDates = msBetweenDates / (24 * 60 * 60 * 1000);
+
+    daysBetweenDates = Math.ceil(daysBetweenDates);
+
+    console.log(daysBetweenDates);
```
Comment (Ruben-Brits) [inline libs/au-pair-dashboard/feature/src/lib/au-pair-dashboard.component.ts:311]: remove console logs please

File: libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.ts [hunk 4]
```diff
@@ -246,16 +256,38 @@ export class EditAuPairProfileComponent implements OnInit {
...
-      this.userDetails.number = val.phone;
-      this.userDetails.address = val.address;
-      this.userDetails.latitude = this.lat;
-      this.userDetails.longitude = this.long;
-      this.userDetails.suburb = this.suburb;
-      this.auPairDetails.payRate = val.payRate;
-      this.auPairDetails.bio = val.bio;
-      this.auPairDetails.experience = val.experience;
-      this.editDetails(this.userDetails, this.auPairDetails);
+      console.log("HUH");
```
Comment (Ruben-Brits) [inline libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.ts:259]: console log

File: libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.ts [hunk 5]
```diff
@@ -246,16 +256,38 @@ export class EditAuPairProfileComponent implements OnInit {
...
+      
+      if(this.errFlag === false)
+      {
+        this.errToast("Please select a valid location from the suggested below.");
+        console.log("NUMBER 1");
```
Comment (Ruben-Brits) [inline libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.ts:264]: console log

File: libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.ts [hunk 6]
```diff
@@ -246,16 +256,38 @@ export class EditAuPairProfileComponent implements OnInit {
...
+      }
+      else if(this.sameFlag === true)
+      {
+        console.log("NUMBER 2");
```
Comment (Ruben-Brits) [inline libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.ts:268]: just check logs please <3

File: libs/parent-dashboard/feature/src/lib/parent-dashboard.ts [hunk 7]
```diff
@@ -243,11 +270,39 @@ export class ParentDashboardComponent implements OnInit{
     }
   }
 
+  async checkResignation()
+  {
+    await this.getParentDetails();
+    await this.getAuPairDetails();    
+
+    console.log("HERE");
```
Comment (Ruben-Brits) [inline libs/parent-dashboard/feature/src/lib/parent-dashboard.ts:278]: console and empty lines

**Review by Ruben-Brits — CHANGES_REQUESTED** [review]

**Review by d-azmanov — APPROVED** [review]

**Review by Ruben-Brits — APPROVED** [review]

**Pull request merged**